cpp/src/arrow/util/bit_util_benchmark.cc

```diff
@@ -159,9 +159,12 @@ static void BenchmarkAndImpl(benchmark::State& state, DoAnd&& do_and) {
 
 static void BenchmarkBitmapAnd(benchmark::State& state) {
   BenchmarkAndImpl(state, [](const internal::Bitmap(&bitmaps)[2], internal::Bitmap* out) {
+    int64_t outPopCount = 0;
     internal::BitmapAnd(bitmaps[0].buffer()->data(), bitmaps[0].offset(),
                         bitmaps[1].buffer()->data(), bitmaps[1].offset(),
-                        bitmaps[0].length(), 0, out->buffer()->mutable_data());
+                        bitmaps[0].length(), 0, out->buffer()->mutable_data(),
+                        &outPopCount);
+    benchmark::DoNotOptimize(outPopCount);
```

Comment:
Augment current benchmark to add PopCount tests. Popcount is not used in many cases.

Reply:
I didn't quite understand, do you mean test for pop count results in bitmap ops tests ? I already added that portion.

Reply:
Sorry I didn't say it clearly. I mean leave current benchmark untouched, and add new benchmarks for `bitmapand+popcount` cases.

Reply:
Done, will push all updates in one go.